### DISCUSSION TITLE
Chamada da API para acesso remoto e inventário "Take Control"

### DISCUSSION THREAD

**Discussion by Givannixs** [amidaware/tacticalrmm#1914] [Q&A]
Pessoal,

Estou integrando o Tactical RMM com uma ferramenta de gerenciamento de chamados e gostaria de "chamar" a API para acesso remoto e inventário dentro dessa nova ferramenta, centralizando tudo. Existe alguma documentação que possa me ajudar com isso?

Agradeço desde já!

**Comment by dinger1986** [comment]
English please

**Comment by Givannixs** [comment]
I'm integrating Tactical RMM with a ticket management tool and would like to call the API for remote access and inventory within this new tool, centralizing everything. Is there any documentation that can help me with this?

**Reply by dinger1986** [reply]
cool, what ticket management tool?

**Reply by Givannixs** [reply]
It is not an open source tool, the name is Desk Manager.

**Reply by dinger1986** [reply]
Is it your software?

**Reply by Givannixs** [reply]
No, it's a paid tool that I currently use to manage my tickets, but it doesn't have inventory and remote access features. We are trying to integrate Tactical RMM with DeskManager.

**Reply by dinger1986** [reply]
Ok, this should help https://docs.tacticalrmm.com/functions/api/

**Reply by Givannixs** [reply]
Can I access MeshCentral features through this API? I ask because we haven't found any reference to this.

**Reply by dinger1986** [reply]
yes, enable swagger and should be able to find it
